lib/broadway/producer.ex

```diff
@@ -179,6 +194,18 @@ defmodule Broadway.Producer do
     {:noreply, [], state}
   end
 
+  def handle_info({RateLimiter, :reset_rate_limiting}, state) do
+    # TODO: to optimize this, we could only take away allowed_messages out of the
```

Comment:
Is it worth doing this? I don't think so.

Reply:
I don't think so either.